### DISCUSSION TITLE
I noticed that material design uses device pixels, how does this package handle that?

### DISCUSSION THREAD

**Discussion by nivle** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3192] [Q&A] [answered]
Hi
I wanted to make a dialog, but then noticed on the [material docs](https://m3.material.io/components/dialogs/specs#6771d107-624e-47cc-b6d8-2b7b620ba2f1) that they use device pixels.

So I was wondering, does this package use device pixels, if so, how?

**Comment by nicolaihenriksen** [comment, marked as answer]
@nivle Perhaps I am misunderstanding your question, but isn't that the default in WPF unless you explicitly set it to something else? See example [docs here](https://learn.microsoft.com/en-us/dotnet/api/system.windows.frameworkelement.height?view=windowsdesktop-7.0#xaml-values) for the `FrameworkElement.Height` property for example.

**Reply by nivle** [reply]
You are right, for some reason I thought the default was in px.
